statement_parser_test.go

```diff
@@ -1601,13 +1601,13 @@ func TestParseClientSideStatement(t *testing.T) {
 		{
 			name:  "Show variable Retry_Aborts_Internally",
 			input: "show variable retry_aborts_internally",
-			want:  "SHOW VARIABLE RETRY_ABORTS_INTERNALLY",
+			want:  "SHOW",
```

Comment:
The (internal) name of the statement changes from the specific `SHOW VARIABLE RETRY_ABORTS_INTERNALLY` to the more generic `SHOW` statement that is the same for all variables.